.gitignore

```diff
@@ -1,4 +1,4 @@
 vendor/
 node_modules/
 TODO.md
-0x_mesh_db/
+0x_mesh/
```

Comment:
Decided to move all files related to Mesh to a single directory to make it easier to clean up. This line in the .gitignore file now matches the new default location.